web-service/src/main/java/com/winsel/dao/entity/Task.java

```diff
@@ -9,80 +12,25 @@
 public class Task {
     @Id
     @GeneratedValue(strategy=GenerationType.IDENTITY)
-    private Integer id;
+    @Getter @Setter private Integer id;
```

Comment:
As each properties have getter and setter you can move the annotation to a class level